src/content/en/updates/2018/12/get-installed-related-apps.md

```diff
@@ -1,12 +1,12 @@
 project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
-description: The getInstalledRelatedApps API is a new web platform API that allows your web app to check to see if your native app is installed on the users device, and vice versa..
+description: The getInstalledRelatedApps() method allows your web app to check whether your native app is installed on a user's device, and vice versa.
```

Comment:
For web.dev, we don't refer to something as a 'Whatever API' unless that's actually a spec name. Since this isn't actually a spec I erred on the side of caution. (Personally, calling a single method or property an API has always felt to me a little like saying 'the royal family' when what we really mean is 'Prince Charles.'